libs/markets/src/lib/components/market-info/market-info-panels.tsx

```diff
@@ -659,6 +671,91 @@ export const LiquidityMonitoringParametersInfoPanel = ({
   return <MarketInfoTable data={marketData} parentData={parentMarketData} />;
 };
 
+export const EthOraclePanel = ({ sourceType }: { sourceType: EthCallSpec }) => {
+  const abis = sourceType.abi?.map((abi) => JSON.parse(abi));
+  const header = 'uppercase my-2 font-bold text-left';
+  return (
+    <>
+      <h3 className={header}>{t('Ethereum Oracle')}</h3>
+      {sourceType.address && (
+        <>
+          <KeyValueTable>
+            <KeyValueTableRow noBorder>
+              <div>{t('Address')}</div>
+              <CopyWithTooltip text={sourceType.address}>
+                <button
+                  data-testid="copy-eth-oracle-address"
+                  className="underline text-right"
+                >
+                  {truncateMiddle(sourceType.address)}
+                  <Icon name="duplicate" className="ml-2" />
+                </button>
+              </CopyWithTooltip>
+            </KeyValueTableRow>
+          </KeyValueTable>
+
+          <div className="my-2">
+            <EtherscanLink address={sourceType.address}>
+              {t('View on Etherscan')}
+            </EtherscanLink>
+          </div>
+        </>
+      )}
+
+      <MarketInfoTable
+        key="eth-call-spec"
+        data={{
+          method: sourceType.method,
+          requiredConfirmations: sourceType.requiredConfirmations,
+        }}
+      />
+      <Accordion>
+        <AccordionPanel
+          itemId="abi"
+          trigger={
+            <AccordionPrimitive.Trigger
+              data-testid="accordion-toggle"
+              className={classNames(
+                'w-full pt-2',
+                'flex items-center gap-2',
+                'group'
+              )}
+            >
+              <div
+                data-testid={`abi-dropdown`}
+                key={'value-dropdown'}
+                className="flex items-center gap-2 w-full"
+              >
+                <div className="uppercase font-bold"> {t('ABI')}</div>
+                <AccordionChevron size={14} />
+                <div className="flex items-center gap-1"></div>
+              </div>
+            </AccordionPrimitive.Trigger>
+          }
+        >
+          <SyntaxHighlighter data={abis} />
+        </AccordionPanel>
+      </Accordion>
+
+      <h3 className={header}>{t('Normalisers')}</h3>
+      {sourceType.normalisers?.map((normaliser, i) => (
+        <MarketInfoTable key={i} data={normaliser} />
+      ))}
+      <h1 className={header}>{t('Filter')}</h1>
```

Comment:
Is this missing some content underneath? We also shouldn't be using h1s here. These should be h3

Reply:
`Filters` are an array of `{ key: any, conditions: any[]}[]`

Reply:
Not sure I follow you. What does the heading 'Filter' relate to? Right now there are two headings of equal importance next to each other. Surely the key heading should be within the map?

![Screenshot 2023-09-28 at 14 42 49](https://github.com/vegaprotocol/frontend-monorepo/assets/6803987/6c0f3ae4-ca0f-4994-bd5a-f6cf46bdcb7f)